open_ventilator_monitor_pi/serial_monitor/serial_monitor_listener.py

```diff
@@ -0,0 +1,80 @@
+from threading import Thread
+
```

Comment:
This looks great, but I recommend we ask Kyle Granat and/or Matthew Allen to review the protocol code.